### PR TITLE
Update Google Vision connector category

### PR DESCRIPTION
**Pull request by AathmanT**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This is to move the Google Cloud Vision connector to the category AI/Google as part of an effort to move all AI connectors to a new category.

One line release note: 
- This will move the Google Cloud Vision connector to the category AI/Google.

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

### CODE REVIEW

**Review by aneeshafedo — DISMISSED** [review]

**Review by abeykoon — DISMISSED** [review]

**Review by aneeshafedo — DISMISSED** [review]

File: openapi/googleapis.vision/Ballerina.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 license = ["Apache-2.0"]
-keywords = ["Business Intelligence/Artificial Intelligence", "Cost/Freemium", "Vendor/Google"]
+keywords = ["AI/Google", "Cost/Freemium", "Vendor/Google"]
 org = "ballerinax"
 name = "googleapis.vision"
 icon = "icon.png"
 distribution = "2201.2.1"
 repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
-version = "1.4.1"
+version = "1.4.2"
```
Comment (LakshanSS) [inline openapi/googleapis.vision/Ballerina.toml:9]: If we have to release this connector with Bal 2201.3.x, we need to bump the connector version to 1.5.0

Comment (AathmanT) [reply openapi/googleapis.vision/Ballerina.toml]: Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/753/commits/c82ce8da6f9d6bbd56a1ed56d8eb103b57261493

Comment (aneeshafedo) [reply openapi/googleapis.vision/Ballerina.toml]: Shall we update the ballerina distribution version too for 2201.3.2

Comment (LakshanSS) [reply openapi/googleapis.vision/Ballerina.toml]: As choreo uses `2201.3.1`, we also use the same version.

@AathmanT Shall we update the bal version in Ballerina.toml and Package.md of this connector.

Comment (AathmanT) [reply openapi/googleapis.vision/Ballerina.toml]: Updated them

**Review by LakshanSS — APPROVED** [review]

**Pull request merged**